graders/java/autograder.sh

```diff
@@ -23,7 +23,8 @@ cp -r /javagrader/*.class /javagrader/org /javagrader/libs/* /grade/classpath
 cp -r /grade/tests/libs/* /grade/classpath 2> /dev/null
 cp -r /grade/serverFilesCourse/java/libs/* /grade/classpath 2> /dev/null
 
-export CLASSPATH="/grade/classpath:$(ls /grade/classpath/*.jar -1 | tr '\n' ':')"
+CLASSPATH="/grade/classpath:$(ls /grade/classpath/*.jar -1 | tr '\n' ':')"
```

Comment:
doing this ensures if the command fails, the exit code of export doesn't shadow it.

Reply:
https://www.shellcheck.net/wiki/SC2155